qa-test-app/src/main/res/drawable/ic_baseline_menu_24.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:height="24dp" android:tint="#000000"
```

Comment:
The menu icon is almost not visible in night mode. Replace `android:tine"#000000"` with
`android:tint="@color/colorOnSurface"`